source/common/upstream/eds.cc

```diff
@@ -62,32 +62,37 @@ void EdsClusterImpl::BatchUpdateHelper::batchUpdate(PrioritySet::HostUpdateCb& h
   const uint32_t overprovisioning_factor = PROTOBUF_GET_WRAPPED_OR_DEFAULT(
       cluster_load_assignment_.policy(), overprovisioning_factor, kDefaultOverProvisioningFactor);
 
+  LocalityWeightsMap empty_locality_map;
```

Comment:
i think this can go next to the line where it's used

Reply:
it's used twice... should I declare it twice separately?

Reply:
my bad, just keep it as is then